sources/oceania/au/act/ACTmapi-Imagery2017.geojson

```diff
@@ -11,10 +11,9 @@
         "license": "CC-BY-4.0",
         "name": "ACTmapi Imagery 2017",
         "available_projections": [
-            "EPSG:4326",
-            "EPSG:28355"
+            "EPSG:3857"
         ],
-        "url": "https://data.actmapi.act.gov.au/arcgis/services/actmapi/imagery2017mga/ImageServer/WMSServer?FORMAT=image/jpeg&VERSION=1.1.1&SERVICE=WMS&REQUEST=GetMap&LAYERS=0&STYLES=&SRS={proj}&WIDTH={width}&HEIGHT={height}&BBOX={bbox}",
+        "url": "https://data.actmapi.act.gov.au/arcgis/rest/services/actmapi/imagery2017mga/ImageServer/exportImage?f=image&format=jpeg&imageSR=3857&bboxSR=3857&bbox={bbox}&size={width},{height}&foo={proj}",
```

Comment:
`foo={proj}` is just to stop make check complaning that {proj} is not found.
We can't use {proj} as the srs param as it will be set as 'EPSG:3857' but we need to pass '3857'.

Reply:
Clever :)

Reply:
Why don't you simply use `{wkid}` ?

Reply:
@don-vip because I don't know which tokens are support :wink: , I would like to help document this, but I'm not sure where to start since ideally we'd only use token supported by all the downstream editors, which means combing through all their sources.

Reply:
OK. I don't know if `{wkid}` is supported by iD and Vespucci. @bhousel @simonpoole